openassessment/xblock/static/sass/xb/elements/_controls.scss

```diff
@@ -41,20 +41,11 @@
     text-decoration: none;
   }
 
-  &.disabled, &[disabled], &.is--disabled {
-    cursor: default;
-    pointer-events: none;
-  }
-
   // CASE: icons in buttons
-  .copy, .icon {
+  .icon {
     display: inline-block;
     vertical-align: middle;
   }
-
-  .copy {
```

Comment:
Button text appears to be fine even without the inner span (and its associated css).